### PR TITLE
MAIN-8108 | Update translations for mercury main.json

### PR DESCRIPTION
**Pull request by kvas-damian**
## Links

* https://wikia-inc.atlassian.net/browse/MAIN-8108

## Description

Update translations for mercury main.json

## Reviewers
@Wikia/x-wing

### CODE REVIEW

**Comment by rogatty** [comment]
Remember about merging it to release-280.

File: front/common/public/locales/en/main.json [hunk 1]
```diff
@@ -150,7 +150,7 @@
     "trending-label": "Trending",
     "user-menu-log-out": "Log out",
     "user-menu-profile": "Profile",
-    "username-anonymous": "A Wikia Contributor",
+    "username-anonymous": "A Wiki Contributor",
```
Comment (rogatty) [inline front/common/public/locales/en/main.json]: It says "A Fandom User" on http://preview.concf.wikia.com/wiki/Special:Contributions/91.102.115.105

Comment (kvas-damian) [inline front/common/public/locales/en/main.json]: Done

File: front/common/public/locales/de/main.json [hunk 2]
```diff
@@ -1,168 +1,168 @@
...
+    "curated-content-editor-edit-community-data-button":"Bearbeiten",
+    "curated-content-editor-edit-main-page":"Hauptseite bearbeiten",
+    "curated-content-editor-editing-header":"Bearbeiten: __label__",
+    "curated-content-editor-empty-section-error":"Abschnitte dürfen nicht leer sein. Bitte füge diesem Abschnitt Einträge hinzu.",
+    "curated-content-editor-enter-category-name":"Gib einen Kategorienamen ein",
+    "curated-content-editor-enter-category-name-tooltip":"Wenn du in Curated Content einen Kategorienamen eingibst, verwende dasselbe Format, das du zur Verlinkung von Inhalten auf Fandom nutzen würdest (z. B. Category:Name_der_Kategorie).",
+    "curated-content-editor-enter-display-name":"Anzeigename eingeben",
+    "curated-content-editor-enter-display-name-tooltip":"Dieser Text wird für diesen Eintrag auf der Hauptseite angezeigt.",
+    "curated-content-editor-enter-page-name":"Seitenname eingeben",
+    "curated-content-editor-enter-page-name-tooltip":"Wenn du in Curated Content einen Seitennamen eingibst, verwende dasselbe Format, das du zur Verlinkung von Inhalten auf Wikia nutzen würdest (z. B. Name_des_Artikels).",
```
Comment (Warkot) [inline front/common/public/locales/de/main.json]: No changes for Curated Content Editor are intended, right? And they will be done in some other iteration, will they?

Comment (kvas-damian) [inline front/common/public/locales/de/main.json]: Will ask.

Comment (Warkot) [inline front/common/public/locales/de/main.json]: Please do. Now, after a second glance, I see that copy in the next line is adjusted to `Wikis` from `Wikias`. So maybe it just slipped through Crowdin.

**Pull request merged**